### PR TITLE
We can stop auto intaking without tripping beam break

### PR DESCRIPTION
**Pull request by oliverstivers**
We have a boolean supplier in auto intake which can be bound to  button, or null if using the command in auto.

### CODE REVIEW

File: src/main/java/frc/robot/commands/AutoIntakeCommand.java [hunk 1]
```diff
@@ -5,18 +5,21 @@
...
 
 public class AutoIntakeCommand extends CommandBase {
     private final CrocodileSubsystem crocodileSubsystem;
     private final double speed;
     private final Timer timer;
     private final Debouncer debouncer;
+    private final BooleanSupplier stopIntake;
 
-    public AutoIntakeCommand(CrocodileSubsystem crocodileSubsystem, double speed) {
+    public AutoIntakeCommand(CrocodileSubsystem crocodileSubsystem, double speed, BooleanSupplier stopIntake) {
```
Comment (stardust4089) [inline src/main/java/frc/robot/commands/AutoIntakeCommand.java:17]: Could this possibly be an overload of the constructor instead?

**Review by stardust4089 — APPROVED** [review]
This will work, but I'd like you to consider my comment.

File: src/main/java/frc/robot/commands/AutoIntakeCommand.java (hunk 1 above)
Comment (oliverstivers) [reply src/main/java/frc/robot/commands/AutoIntakeCommand.java:17]: Yes I can change that

**Comment by stardust4089** [comment]
Looks like some stuff got messed up here with your merges, a branch got merged that wasnt supposed to. Closing due to that.

**Pull request closed without merging**